requirements.txt

```diff
@@ -6,4 +6,5 @@ streamlit
 watchdog
 boto3
 python-dotenv
-geonamescache
\ No newline at end of file
+geonamescache
+psycopg2
```

Comment:
Can this requirements.txt file be placed within the dashboard directory

Reply:
done!